.github/workflows/build_on_merge.yml

```diff
@@ -1,9 +1,10 @@
-name: Push to master build
+name: Build on merge to latest
 
 on:
   push:
     branches:
-      - master
+      - latest
+      - master # for bc only
```

Comment:
In this repo, it can be completely dropped - no BC on this tool, just on the main repo